slicer_configurations/README.md

```diff
@@ -23,11 +23,12 @@ like so:
 
 ---
 
-| Creator | Config title | Description | V0 | V1.8 | V2.4 | VSW |
-| --- | --- | --- | --- | --- | --- | --- |
-| hartk1213 | [PrusaSlicer V0 Bed and Textures](./PrusaSlicer/hartk1213/V0) | Bed STL and textures for Prusa Slicer | :heavy_check_mark: | :x: | :x: | :x: |
-| hijax_pl_ | [Simplify3D](./Simplify3D/hijax_pl/beds) | Bed images for Simplify3D | :x: | :x: | :heavy_check_mark: | :x: |
-| hoangnam123 | [SuperSlicer v2.4 Bed](./SuperSlicer/hoangnam123) | Bed STL for SuperSlicer | :x: | :x: | :heavy_check_mark: | :x: |
-| koonweee | [SuperSlicer v1.8 Bed](./SuperSlicer/koonweee/beds) | Bed STL for SuperSlicer | :x: | :heavy_check_mark: | :x: | :x: |
+| Creator     | Config title                                                  | Description                            | V0                 | V1.8               | V2.4               | VSW |
```

Comment:
This table cleanup not strictly necessary.

I can revert these changes if the noisy diff is obnoxious.

Reply:
Yeah, don't do that, the next person that adds an entry is not going to care about alignment, and the table will be out of alignment again anyway.  As much as I personally would prefer the table to be aligned, it's not going to remain that way.

Reply:
Fixed. Yeah, I thought the same thing after writing this.